static/monorail-2.0.yaml

```diff
@@ -556,6 +584,74 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  /skus/{identifier}/pack:
+    x-swagger-router-controller: skus
+    put:
+      operationId: skusIdPutPack
+      x-privileges: [ 'write' ]
+      x-authentication-type: [ 'jwt' ]
+      summary: |
+        post a sku pack to specific sku
+      description: |
+        post a sku pack to specific sku
+      parameters:
+        - name: identifier
+          in: path
+          description: |
+            The profile name.
+          required: true
+          type: string
+      tags: [ "/api/2.0" ]
+      consumes:
+        - application/x-www-form-urlencoded
+      responses:
+        200:
+          description: |
+            return nodes associated with that sku
+          schema:
+            type: object
+        404:
+          description: |
+            There is no sku with identifier.
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    delete:
+      operationId: skusIdDeletePack
+      x-privileges: [ 'write' ]
+      x-authentication-type: [ 'jwt' ]
+      summary: |
+        Delete a pack for a specific sku
+      description: |
+        Delete a pack for a specific sku.
+      parameters:
+        - name: identifier
+          in: path
+          description: |
+            The profile name.
+          required: true
+          type: string
+      tags: [ "/api/2.0" ]
+      consumes:
+        - application/x-www-form-urlencoded
+      responses:
+        200:
+          description: |
+            return nodes associated with that sku
```

Comment:
Hi Andrew it is in the sku-pack-service.js


  SkuPackService.prototype.getNodesSkusById = function(id) {
        return waterline.skus.needByIdentifier(id)
            .then(function (sku) {
            return waterline.nodes.find({ sku: sku.id });
        });
    };